Expecta/Expecta.h

```diff
@@ -12,3 +12,4 @@ FOUNDATION_EXPORT const unsigned char ExpectaVersionString[];
 
 // Enable shorthand by default
 #define expect(...) EXP_expect((__VA_ARGS__))
+#define expect_fail(...) EXP_expect_fail((__VA_ARGS__))
```

Comment:
`fail` felt like the `#define` was going to mangle with other people's block names. Think we can get away with it? Happy to switch, prefer that name.